# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/metamail/25_mm-2.7-sunquote.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- mm2.7/src/bin/sun-message.csh~	Tue May 19 10:22:52 1998
+++ mm2.7/src/bin/sun-message.csh	Tue May 19 10:26:39 1998
@@ -5,6 +5,8 @@
 # This is sun-message.  It looks at $2 to figure out how to decode $1, then gives the
 # user a short menu of choices, display, save, or quit.
 #
+# $2 is untrustworthy data. Quote it.
+#
 mkdir /tmp/decode.$$
 cd /tmp/decode.$$
 
@@ -12,7 +14,7 @@
     set METAMAIL_PAGER=more
 endif
 
-if ($2 == "uuencode") then
+if ("$2" == "uuencode") then
     set fn=`mktemp $METAMAIL_TMPDIR/audio.XXXXXX`
     test -n "$fn"
     if $? then
